lib/classes/Swift/MemorySpool.php

```diff
@@ -16,6 +16,7 @@
 class Swift_MemorySpool implements Swift_Spool
 {
     protected $messages = array();
+    protected $flushRetries = 3;
```

Comment:
This should be private instead (all new properties are added as private now, to avoid increasing the BC surface, even though lots of older properties still exist as protected for BC reasons)